plugins/aws/workspaces/workSpacesOperationalState.js

```diff
@@ -0,0 +1,60 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'WorkSpaces Healthy Instances',
```

Comment:
can you please make sure that file name is same as title, WorkSpace Healthy Instance looks fine